packages/dds/sequence/src/test/generateSharedStrings.ts

```diff
@@ -114,3 +115,99 @@ export function* generateStrings(): Generator<[string, SharedString]> {
         yield [`${version}/withIntervals`, sharedString];
     }
 }
+
+// Adding a flag to determine if we should run a test on the document in snapshotVersion.spec.ts.
+// We only want to test the wiithIntervals.json documents, since these are
+// the only ones with format changes.
+export function* generateTestStrings(): Generator<[string, TestSharedString, boolean]> {
```

Comment:
It would be good to make the return type of this an object rather than an array, that way you can name the fields in code and have a clearer place to document expected behaviors. For example the object type
`{ snapshotPath: string; expected: SharedString; snapshotIsNormalized boolean }` would give more readable code and you can document fields on that type, which gives nice intellisense.